src/core/mod.rs

```diff
@@ -24,7 +24,7 @@ use chainstate::burn::BlockHeaderHash;
 use util::log;
 
 // fork set identifier -- to be mixed with the consensus hash (encodes the version)
-pub const SYSTEM_FORK_SET_VERSION : [u8; 4] = [22u8, 0u8, 0u8, 0u8];
+pub const SYSTEM_FORK_SET_VERSION : [u8; 4] = [23u8, 0u8, 0u8, 0u8];
 
 // p2p network version 
 pub const PEER_VERSION : u32 = 0x16000000;      // 22.0.0.0
```

Comment:
should this increment with `SYSTEM_FORK_SET_VERSION`?

Reply:
Yes, thanks.